debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/dialect/postgres/TsvectorType.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.jdbc.dialect.postgres;
+
+import java.util.List;
+
+import org.apache.kafka.connect.data.Schema;
+
+import io.debezium.connector.jdbc.type.AbstractType;
+import io.debezium.sink.column.ColumnDescriptor;
+import io.debezium.sink.valuebinding.ValueBindDescriptor;
+
+/**
+ * An implementation of {@link AbstractType} for the PostgreSQL {@code tsvector} data type.
+ *
+ * A {@code tsvector} is a full-text search data type in PostgreSQL. It stores lexeme vectors—
+ * preprocessed, searchable representations of textual content—used to support efficient
+ * full-text search queries.
+ *
+ * The PostgreSQL connector serializes {@code tsvector} data using the Debezium semantic type
+ * {@link io.debezium.data.TsVector}, which represents the structured content of a tsvector field.
+ *
+ * @author Pranav Tiwari
+ */
+public class TsvectorType extends AbstractType {
+
+    public static TsvectorType INSTANCE = new TsvectorType();
+
+    @Override
+    public String getTypeName(Schema schema, boolean isKey) {
+        return "tsvector";
+    }
+
+    @Override
+    public String[] getRegistrationKeys() {
+        return new String[]{ "io.debezium.data.Tsvector" };
```

Comment:
```suggestion
        return new String[]{ io.debezium.data.TsVector.LOGICAL_NAME };
```
Since the `TsVector` type was added to `debezium-core`, we can use the logical name directly.